### ISSUE TITLE
Add auto update support for appimage releases

### ISSUE THREAD

**Issue by mborgerson** [xemu-project/xemu#1349]
### Feature Request

Now that xemu has appimage releases, it makes sense to support updating the appimage as xemu does on Windows, in the event the user does not have/want xemu to be managed by other software.

**Comment by davidhedlund** [comment]
Agreed. This feature is already implemented in other emulator projects, including:
- Cemu: Help → Check for updates
- PCSX2: Help → Check for Updates
- RPCS3: Help → Check for Updates
- [shadPS4QtLauncher](https://github.com/shadps4-emu/shadps4-qtlauncher/releases): Help → Check for Updates

**Comment by davidhedlund** [comment]
quick-sharun is a wrapper over `lib4bin`/`sharun` that greatly simplifies deployment, it also offers additional features and fixes. ([Including an updater](https://github.com/pkgforge-dev/Anylinux-AppImages/blob/main/useful-tools/hooks/self-updater.hook))
